databricks/koalas/tests/test_dataframe.py

```diff
@@ -2923,8 +2923,8 @@ def test_drop_duplicates(self):
 
     def test_reindex(self):
         index = pd.Index(["A", "B", "C", "D", "E"])
-
-        pdf = pd.DataFrame({"numbers": [1.0, 2.0, 3.0, 4.0, None]}, index=index)
+        columns = pd.Index(["numbers"], name="cols")
```

Comment:
Shall we reserve one test case for "when the original column is not named"?

Reply:
Yes, I also added a test reindexing with a non named column.

Reply:
Great!